docs/api/index.rst

```diff
@@ -14,7 +14,7 @@ Changelog
 1.10 (unreleased)
 '''''''''''''''''
 
-- Add full-text case insensitive filtering on plural endpoints (e.g ``?like_person=Tim``)
+- Add text case insensitive filtering on plural endpoints (e.g ``?like_person=Tim``)
```

Comment:
I personally think full-text isn't wrong here, but if we're going to change it, I think we should include a stronger word than "filtering". How about "Add pattern-matching query to filtering on plural endpoints" or "Add substring query to filtering on plural endpoints"?